src/pages/HomeScreen/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from "react";
+import { getValueDB, setValueDB, updateValueDB } from "../../firebase";
+
+interface News {
```

Comment:
it looks like a reusable interface need to create file with types

`News, User `